### DISCUSSION TITLE
v1.3

### DISCUSSION THREAD

**Discussion by HenryNdubuaku** [cactus-compute/cactus#259] [Announcements]
## What's Changed
* Apple NPU support by @HenryNdubuaku 
* Optimized softmax to use Horner's method by @ParkiratS 
* Tunix finetuning by @ncylich 
* PCM input stream to whisper transcription by @devabhixda 
* Cli mobile benchmarks by @jakmro 
* Telemetry framework by @devabhixda 
* Python bindings by @HenryNdubuaku 

Bindings @devabhixda @jakmro @mhayes853 
- The CLI has changed a bit, read the cactus_ffi.h files properly. 
- If you need keys for the Pro, please reach out to henry@cactuscompute.com 

**Full Changelog**: https://github.com/cactus-compute/cactus/compare/v1.2...v1.3

<hr /><em>This discussion was created from the release <a href='https://github.com/cactus-compute/cactus/releases/tag/v1.3'>v1.3</a>.</em>
